.travis.yml

```diff
@@ -5,8 +5,20 @@ sudo: false
 addons:
   apt:
     sources: &zeromq_source
```

Comment:
This does not appear to be referenced anymore in your changes.

Reply:
Yeah, I forgot it there